### ISSUE TITLE
Can it support the FIND functionality?

### ISSUE THREAD

**Issue by superzilong** [lkytal/BuiltinCmd#14]
Sometimes it needs to find the keyword of the output text. It will be very convenient if it can search in this buildin command window.

**Comment by astrohart** [comment]
Yes, me too.  I would like to also have a tabbed window with PowerShell, CMD, Git, etc. like:

![image](https://user-images.githubusercontent.com/8261875/188223204-da701390-8adb-44d3-9699-fd3991344eb0.png)

(see [this extension](https://marketplace.visualstudio.com/items?itemName=GrzegorzKozub.ConsoleLauncher))
